github/GithubIntegration.py

```diff
@@ -212,18 +218,34 @@ def create_jwt(self, expiration: int | None = None) -> str:
         return self.auth.create_jwt(expiration)
 
     def get_access_token(
-        self, installation_id: int, permissions: dict[str, str] | None = None
+        self,
+        installation_id: int,
+        permissions: dict[str, str] | None = None,
+        repositories: list[str | int] | None = None,
     ) -> InstallationAuthorization:
         """
         :calls: `POST /app/installations/{installation_id}/access_tokens <https://docs.github.com/en/rest/apps/apps#create-an-installation-access-token-for-an-app>`
         """
         if permissions is None:
             permissions = {}
 
+        if repositories is not None and not (isinstance(repositories, list) and all(isinstance(r, (str, int)) for r in repositories)):
```

Comment:
The unit tests fails here because in Python, booleans are subclass of integers 🙃 

https://docs.python.org/3/c-api/bool.html